sulenchy/MyDiary-React-Redux
# Tie out modal component

### What does this PR do?

- Tie out the modal and sidebar component

### Description of Task to be completed?

- Add modal component
- Add sidebar component
- update the styles

### How should this be manually tested?

- Check out this branch
- Do `npm i` to install the packages
- Do `npm run start:dev`
- Test the app in chrome

### Any background context you want to provide?

### What are the relevant pivotal tracker stories?

### Screenshots (if appropriate)

### Questions:
